### PR TITLE
CM-389: Add defaultDescription

### PR DESCRIPTION
**Pull request by ayumi-oxd**
### Jira Ticket:
CM-389

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-389

### Description:
- Added new field `defaultDescription` to `activityTypes` and `facilityTypes`
- Displayed `defaultDescription` on the park and site pages if there's no `description` of `parkActivities` and `parkFacilities`

### CODE REVIEW

File: src/staging/src/templates/park.js [hunk 1]
```diff
@@ -519,6 +519,7 @@ export const query = graphql`
           icon
           iconNA
           rank
+          defaultDescription
```
Comment (molund) [inline src/staging/src/templates/park.js:522]: Does this need to be on the site template too?

Comment (ayumi-oxd) [reply src/staging/src/templates/park.js:522]: I believe so. For example, this site [Wells Gray Park: Murtle Lake](https://bcparks.ca/wells-gray-park/murtle-lake/) has an empty kayak accordion.

Comment (ayumi-oxd) [reply src/staging/src/templates/park.js:522]: I believed I DID but I DIDN'T 😅 Updated the site too.

File: src/staging/src/components/park/parkFacility.js [hunk 2]
```diff
@@ -100,7 +100,11 @@ export default function ParkFacility({ data }) {
               </Accordion.Toggle>
               <Accordion.Collapse eventKey={"parkFacility" + index}>
                 <div className="p-4">
-                  <HtmlContent>{facility.description}</HtmlContent>
+                  <HtmlContent>
+                    {facility.description ?
```
Comment (molund) [inline src/staging/src/components/park/parkFacility.js:104]: this should use isNullOrWhiteSpace

File: src/staging/src/components/park/parkActivity.js [hunk 3]
```diff
@@ -99,7 +99,11 @@ export default function ParkActivity({ data }) {
               </Accordion.Toggle>
               <Accordion.Collapse eventKey={"parkActivity" + index}>
                 <div className="p-4">
-                  <HtmlContent>{activity.description}</HtmlContent>
+                  <HtmlContent>
+                    {activity.description ?
```
Comment (molund) [inline src/staging/src/components/park/parkActivity.js:103]: this should use isNullOrWhiteSpace

**Comment by molund** [comment]
This worked for an activity (canoeing) when I tested it, but it did not work for a facility (winter camping). This is probably due to not checking isNullOrWhitespace

**Comment by ayumi-oxd** [comment]
> This worked for an activity (canoeing) when I tested it, but it did not work for a facility (winter camping). This is probably due to not checking isNullOrWhitespace

It didn't work because I didn't set the description operator for the camping 😬
Updated `campingDetails` and added isNullOrWhitespace to check

**Review by molund — APPROVED** [review]

**Review by foley-lynn — APPROVED** [review]

**Pull request merged**